### ISSUE TITLE
Quick Save / Load

### ISSUE THREAD

**Issue by hi-names-nat** [Geeknerd1337/NSCM#34]
Source and GoldSRC have systems in place to allow the user to save and load with the press of a single button (F6 and F7). This is a feature we should consider for NSCM.

**Comment by Geeknerd1337** [comment]
@hi-names-nat Do you know how a technical implementation of this system would look like?

**Comment by hi-names-nat** [comment]
@Geeknerd1337 I will do some prototyping and get back to you on how I imagine a prototype of this system would look.

**Comment by hi-names-nat** [comment]
This is a project. We should consider purchasing a save system.

**Comment by Geeknerd1337** [comment]
@hi-names-nat I would not be against this if its proving to be that difficult.

**Comment by hi-names-nat** [comment]
@Geeknerd1337 We absolutely could build our own system, but we have to ask the question **is it worth it?** I'm going to research save systems and such.

**Comment by Geeknerd1337** [comment]
@hi-names-nat This is true, please tell me what your research shows. If it's a fairly simple endeavor, like not super complex then it's probably not worth paying for it. If it turns out to be extra mundo hard then we'll look into buying one. Out of curiosity do you know how half-life handles it?
